images/kyverno-background-controller/README.md

```diff
@@ -0,0 +1,12 @@
+<!--monopod:start-->
+# kyverno-background-controller
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/kyverno-background-controller` |
+| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/kyverno-background-controller.svg) |
```

Comment:
We made some changes to `monopod readme` to remove these lines. Can you update your monopod (`$(cd monopod && go install)`) and regenerate these?